templates/common_header.php

```diff
@@ -54,25 +54,36 @@ function addss(d,u,l) {
 
     <?php if (OPTION_WEB_DOMAIN == 'writetothem.com'): ?>
 
+    <script defer>Object.defineProperty(document,"cookie",{get:function(){var t=Object.getOwnPropertyDescriptor(Document.prototype,"cookie").get.call(document);return t.trim().length>0&&(t+="; "),t+="_ga=GA1.1."+Math.floor(1e9*Math.random())+"."+Math.floor(1e9*Math.random())},set:function(t){t.trim().startsWith("_ga")||Object.getOwnPropertyDescriptor(Document.prototype,"cookie").set.call(document,t)}});</script>
+    <script defer src="https://www.googletagmanager.com/gtag/js?id=G-9ER8NZ5405"></script>
     <script>
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-        ga('create', '<?=OPTION_GOOGLE_ANALYTICS_TRACKING_CODE?>', 'writetothem.com', {'storage':'none'});
-        ga('set', 'anonymizeIp', true);
-        ga('send', 'pageview');
+        var client_id = Math.floor(Math.random() * 1000000000) + '.' + Math.floor(Math.random() * 1000000000);
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'G-9ER8NZ5405', {'client_id': client_id, 'cookie_expires': 1});
```

Comment:
Why is this hard coded instead of using `<?=OPTION_GOOGLE_ANALYTICS_TRACKING_CODE?>`?

Reply:
We're inconsistent on this in different services. Ultimately there's no good reason, in codebases that are only used in one place (in this case it's behind a 'is this writetothem.com' check) not to hardcode something that's going to be public anyway.

The real practical problem is I have less experience in changing these variables behind the scenes, and this would make the deploy very slightly more difficult for *me* to do. Obvs this is a solvable problem if we really want to keep it as a variable.